Week2/Section2_w2/MySpotify/PlaylistContainer.cpp

```diff
@@ -0,0 +1,46 @@
+#include "PlaylistContainer.h"
+#include <iostream>
+
+PlaylistContainer::PlaylistContainer()
+{
+	PlaylistCount = 0;
+}
+
+void PlaylistContainer::AddPlaylist(Playlist NewPlaylist)
+{
+	if (PlaylistCount < MaxPlaylistCount)
+	{
+		PlaylistArray[PlaylistCount] = NewPlaylist;
+		PlaylistCount++;
+	}
+}
+
+Playlist PlaylistContainer::GetPlaylist(int Index) const
+{
+	if (Index < MaxPlaylistCount)
+	{
+		return PlaylistArray[Index];
+	}
+	else
+	{
+		return PlaylistArray[PlaylistCount];
```

Comment:
What if PlaylistCount was equal to MaxPlaylistCount?

Reply:
that doesn't happend because...
```
void PlaylistContainer::AddPlaylist(Playlist NewPlaylist)
{
	if (PlaylistCount < MaxPlaylistCount)
	{
		PlaylistArray[PlaylistCount] = NewPlaylist;
		PlaylistCount++;
	}
}
```

when you try to exceed MaxPlaylistCount it doesn't create a new playlist, but I realize that you can keep "creating" playlist with songs, it does nothing. I'll add and error message for that 👍